app/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "app",
+  "version": "0.0.1",
+  "description": "Learn about what Sustainable Means and create Docs you need to Prove it",
+  "productName": "SFOSC Wizard",
+  "cordovaId": "org.cordova.quasar.app",
+  "author": "Daniel Thompson-Yvetot <35242872+nothingismagick@users.noreply.github.com>",
+  "private": true,
+  "scripts": {
+    "lint": "eslint --ext .js,.vue src",
+    "test": "echo \"See package.json => scripts for available tests.\" && exit 0",
+    "test:unit": "jest --updateSnapshot",
+    "test:unit:coverage": "jest --coverage",
+    "test:unit:watch": "jest --watch",
+    "test:unit:watchAll": "jest --watchAll",
+    "serve:test:coverage": "quasar serve test/jest/coverage/lcov-report/ --port 8788",
```

Comment:
I personally prefer to avoid global installs (`npm install -g @quasar/cli` in the readme in this case)
Considering this serve also depends on the CLI, can we add these scripts?

```
"start": "quasar dev",
"build": "quasar build",
```